.nvmrc

```diff
@@ -1 +1 @@
-12.16
\ No newline at end of file
+12.16.0
```

Comment:
I have 12.16.3 locally, isn't this going to mess with minor versions?

Reply:
I will remove this. I was trying to fix the broken build.